docs/getting-started/first-abn.md

```diff
@@ -55,44 +55,24 @@ A sample application using the Iter8 SDK is provided. Deploy both the frontend a
 
 ## Describe the application
 
-In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A `ConfigMap` is used to describe the make up of possible versions:
+In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A _routemap_ is created to do this. A routemap contains a description of each version of an application and may contain a routing template. To create the routemap:
```

Comment:
"and may contain a routing template"

Is it better to say "routing template" or "routing section"? It might be nice to include a hyperlink for a user to learn more about routing, if possible.